### PR TITLE
Add astroid

### PR DESCRIPTION
**Pull request by jakirkham**
Generated with `conda skeleton pypi` and cleaned up after.

### CODE REVIEW

File: recipes/astroid/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - python
+    - setuptools
+    - six
+    - lazy_object_proxy
+    - wrapt
+
+  run:
+    - python
+    - six
+    - lazy_object_proxy
```
Comment (jakirkham) [inline recipes/astroid/meta.yaml]: Needs to be `lazy-object-proxy`.

File: recipes/astroid/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
+    - lazy_object_proxy
```
Comment (jakirkham) [inline recipes/astroid/meta.yaml]: Needs to be `lazy-object-proxy`.

**Comment by jakirkham** [comment]
Added your change here, @ericdill. Thanks for pointing that out to me. 😄

**Comment by ericdill** [comment]
Ah yes the astroid PR is a much more sensible place to update the astroid meta.yaml... :laughing:

**Comment by jakirkham** [comment]
No worries. Thanks for the change.

**Comment by jakirkham** [comment]
Any more feedback on this?

**Comment by jakirkham** [comment]
Merging as I haven't heard anything else that needs to be done. Hope that is ok.

**Comment by jakirkham** [comment]
Would you be interested in helping maintain, @ericdill?

**Comment by ericdill** [comment]
Yes please

**Comment by jakirkham** [comment]
Adding in PR ( https://github.com/conda-forge/astroid-feedstock/pull/1 ).

**Comment by jakirkham** [comment]
@PCManticore @sthenault @shlomme @ceridwen @jcristau @The-Compiler, just letting you know that we have added `astroid` to [conda-forge]( http://conda-forge.github.io/ ). This will allow users to easily install this with [`conda`]( http://conda.pydata.org/docs/ ) ( [`conda install -c conda-forge astroid`]( http://anaconda.org/conda-forge/astroid ) ). The maintenance burden here is light as all releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe in [this feedstock (repo)]( https://github.com/conda-forge/astroid-feedstock ). If any of you have any interest in being added as a maintainer (to make new releases for instance), we would be more than happy to add any of you to the maintainers list. Thanks for your work on this great

**Pull request merged**